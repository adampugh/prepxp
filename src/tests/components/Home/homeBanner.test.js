import React from "react";
import { shallow } from "enzyme";

import HomeBanner from "../../../components/Home/homeBanner";

describe("<HomeBanner />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<HomeBanner />);
        expect(wrapper).toMatchSnapshot();
    });
});