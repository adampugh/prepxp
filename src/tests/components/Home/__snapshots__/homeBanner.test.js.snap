// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeBanner /> should match snapshot 1`] = `
<section
  className="homeBanner"
>
  <div
    className="container"
  >
    <div
      className="homeBanner__content"
    >
      <div
        className="homeBanner__content__textPanel"
      >
        <div
          className="homeBanner__content__textPanel__title"
        >
          <h1>
            Prepare for success
          </h1>
        </div>
        <div>
          <p>
            Prepare for job interviews by answering pracitcal interview questions! Create and build your own lists of interview questions or import from curated lists.
          </p>
          <Link
            replace={false}
            to="/prepxp/login"
          >
            <button
              className="btn"
            >
              Get Started
            </button>
          </Link>
        </div>
      </div>
      <div
        className="homeBanner__content__picturePanel"
      >
        <div>
          <img
            alt="man sitting at a desk"
            className="bannerText__image"
            src="deskimg.png"
          />
        </div>
      </div>
    </div>
  </div>
</section>
`;
